clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1255,6 +1257,7 @@ class raw2trace_t {
     uint64 count_rseq_side_exit_ = 0;
     uint64 earliest_trace_timestamp_ = (std::numeric_limits<uint64>::max)();
     uint64 latest_trace_timestamp_ = 0;
+    uint64 final_trace_instr_count_ = 0;
```

Comment:
How about a sanity unit test in raw2trace_unit_tests with a couple of threads.  Looking: surprisingly there is no test at all of any of these stats.  Would be good to add something to at least exercise this.

Reply:
Done. Would be good to do a re-review for the new test code.